sci-libs/fftw/fftw-3.3.8.recipe

```diff
@@ -63,33 +95,86 @@ BUILD_PREREQUIRES="
 	"
 
 defineDebugInfoPackage fftw$secondaryArchSuffix \
-	$libDir/libfftw3.so.$libVersion \
-	$libDir/libfftw3f.so.$libVersion
+	"$libDir"/libfftw3.so.$libVersion \
+	"$libDir"/libfftw3_threads.so.$libVersion \
+	"$libDir"/libfftw3f.so.$libVersion \
+	"$libDir"/libfftw3f_threads.so.$libVersion \
+	"$libDir"/libfftw3l.so.$libVersion \
+	"$libDir"/libfftw3l_threads.so.$libVersion \
+	"$libDir"/libfftw3q.so.$libVersion \
+	"$libDir"/libfftw3q_threads.so.$libVersion
 
 BUILD()
 {
-	make clean && make distclean || true
+	mkdir -p double long-double quad float
+
+	CONFIGURE="--enable-shared --enable-threads"
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	CONFIGURE_X86="--enable-sse2 --enable-avx --enable-avx2 --enable-avx512"
+fi
 
 	# build double-precision version
-	runConfigure ./configure --enable-shared
+	cd double
+	runConfigure ../configure \
+		$CONFIGURE \
+		$CONFIGURE_X86
+	make $jobArgs
+	cd ..
+
+	# build long-double-precision version
+	cd long-double
+	runConfigure ../configure \
+		--enable-long-double \
+		$CONFIGURE
 	make $jobArgs
-	make install
+	cd ..
```

Comment:
Two things in this comment:
* A suggestion to use `pushd long-double` and `popd` instead of `cd long-double` and `cd ..`.
* Is it on purpose that you are not using `$CONFIGURE_X86` for `long-double`, although you are using it for `double`? Important: I don't know if using it for `long-double` would be appropriate. I"m just asking.

```diff
 	# build long-double-precision version
-	cd long-double
+	pushd long-double
 	runConfigure ../configure \
 		--enable-long-double \
- 		$CONFIGURE
+ 		$CONFIGURE \
+		$CONFIGURE_X86
 	make $jobArgs
-	cd ..
+	popd
```
BTW, if you like the `pushd` & `popd` then you can use them for the other blocs too.

Reply:
Only single and double can handle sse2.

Reply:
Ok, thanks.